utils/config.ts

```diff
@@ -9,7 +9,14 @@ import * as t from "io-ts";
 import { IntegerFromString } from "italia-ts-commons/lib/numbers";
 import { readableReport } from "italia-ts-commons/lib/reporters";
 import { NonEmptyString } from "italia-ts-commons/lib/strings";
-import { withDefault } from "italia-ts-commons/lib/types";
+import { enumType, withDefault } from "italia-ts-commons/lib/types";
+
+export enum NHPartitionFeatureFlag {
+  "none" = "none",
+  "beta" = "beta",
+  "canary" = "canary",
+  "all" = "all"
+}
```

Comment:
> _Enums are great, but have you tried literal unions?_

```suggestion
export type NHPartitionFeatureFlag = t.TypeOf<typeof NHPartitionFeatureFlag>;
export const NHPartitionFeatureFlag = t.union([
  t.literal("none"),
  t.literal("beta"),
  t.literal("canary"),
  t.literal("all")
]);
```

Although they serve the same purpose, unions are way more powerful than enums. [Here](https://fettblog.eu/tidy-typescript-avoid-enums/) some thoughts on the subject, but the most important for us is: they better support exhaustive checks 🚀

Reply:
Done as `keyof` in [888e53a](https://github.com/pagopa/io-functions-pushnotifications/pull/17/commits/888e53a33e5c65b25e5e322287cd8407a9389747)